### ISSUE TITLE
 Need to change Nav Bar position 

### ISSUE THREAD

**Issue by sathwikrao4545** [sathwikrao4545/IS601grp#11]
The position of the navigation bar should be adjusted upwards for improved layout and accessibility.

**Comment by sathwikrao4545** [comment]
Nav bar Position Updated.
Before Updation
<img width="577" alt="beforenavpos" src="https://github.com/sathwikrao4545/IS601grp/assets/92375656/c59b133c-dea8-474f-9f19-974e6b279ced">

After Updation
<img width="601" alt="afternavpos" src="https://github.com/sathwikrao4545/IS601grp/assets/92375656/8263c60f-b37e-495a-8851-d7f0ab186a44">
